avocado/plugins/testlogs.py

```diff
@@ -63,6 +76,12 @@ def post(self, job):
                 except (FileNotFoundError, PermissionError) as error:
                     LOG_UI.error('Failure to access log file "%s": %s',
                                  path, error)
+        if summary_tests:
+            LOG_UI.info("")
+            LOG_UI.info(" Test summary:")
+            for test in summary_tests:
```

Comment:
As you can see from the CI results, these lines are ending up in the TAP, json, xml files.

Reply:
Actually the files are right. Only the stdout is wrong. I'm fixing, thanks.